### DISCUSSION TITLE
Open feed directly on tap of header

### DISCUSSION THREAD

**Discussion by MasothU** [tughi/aggregator-android#93] [Ideas]
Another request from my side with which I could already now fully hop on the new version ☺️
On the old version I could tap on the header (or the icon next to it) and the feed opened directly in the browser, without having to open it (see screenshot)
![Screenshot_20240107_064205_Aggregator](https://github.com/tughi/aggregator-android/assets/17427639/1815a054-f925-46fd-84ad-c18c8eed5693)

Thanks in advance for for your great work. Really appreciate it.

**Comment by tughi** [comment]
This is a duplicate of #75
